ush/python/pygfs/task/gfs_forecast.py

```diff
@@ -7,7 +7,7 @@
 from pygw.task import Task
 from pygfs.ufswm.gfs import GFS
 
-logger = logging.getLogger(__name__.split('.')[-1])
+logger = logging.getLogger(__name__.split(".")[-1])
```

Comment:
I will remember to use `"` next time onwards so that this does not show up as a change from others.
Though, then we may need to make it a rule for everyone.

Reply:
@aerorahul This is another artifact from `black`. No need to enforce, we can just push it through the `black` linter  pre-commit. Also, not need to bother others. I can make these changes incrementally in PRs.